app/src/main/java/de/westnordost/streetcomplete/screens/about/AboutFragment.kt

```diff
@@ -86,15 +72,20 @@ class AboutFragment : TwoPaneListFragment(), HasTitle {
         return installerPackageName == "com.android.vending"
     }
 
-    private fun openUrl(url: String): Boolean {
-        val intent = Intent(Intent.ACTION_VIEW, url.toUri())
-        tryStartActivity(intent)
-        return true
+    private fun setUrlClickListener(preferenceKey: String, url: String) {
+        setUrlClickListener(findPreference(preferenceKey), url)
+    }
+
+    private fun setUrlClickListener(preference: Preference?, url: String) {
+        preference?.setOnPreferenceClickListener {
+            openUri(url)
+            true
+        }
```

Comment:
I think this would be better / more readable:

```
    private fun Preference.setClickOpensUrl(url: String) {
        setOnPreferenceClickListener {
            openUri(url)
            true
        }
```

or even drop these completely.